### ISSUE TITLE
Windows Admin Center docs Needs clarification?

### ISSUE THREAD

**Issue by shelterx** [MicrosoftDocs/windowsserverdocs#5392]

**Comment by illfated** [comment]
I notice that you have formed the issue ticket as one or more questions.
Please note that the MicrosoftDocs repository is intended for document improvements.

Based on my own experience, it is recommended to use Microsoft forum pages/social pages for Q & A, so allow me to add a canned post I have composed during the last couple of years I have been participating in this community:

> You may want to look elsewhere for answers, unless you have suggestions for improving one or more specific pages in the MS Docs repository.
>
> Disclaimer: I am not affiliated with Microsoft or the Microsoft Docs teams here on GitHub.
>
> The intention of this repository is to improve on the existing documentation by providing the users a way to report factual errors, improvement suggestions and request updates for new versions.
> The MS Docs team members might not have time to answer your question, both due to the small size of the team and the big workload.
>
> Try the following resources to get in touch with users who may be able to help you:
> - TechNet forums: https://social.technet.microsoft.com/Forums/en-US/home
> - MSDN: https://social.msdn.microsoft.com/Forums/en-US/home
> - Microsoft Community: https://answers.microsoft.com/

**Comment by shelterx** [comment]
Yes, it ended up more like a question after the edit. 
But the documentation is very unclear about what to expect.
Maybe they could add it to the known issues part of the documentation.
"Remote desktop extension currently does not support Smart Card login" (or SSO if that's correct)
"Powershell extension requires you to enter credentials when accessing the Admin Center web from another server than from the WAC gateway itself"

I'm not sure if the above is correct but it seems like it.

**Comment by illfated** [comment]
Fair enough, although making changes to the documentation requires a fair bit of involvement in each case. One place to start would be to post suggestions for specific changes by opening a ticket for each page in question, by using the button **[This page]** at the bottom of the page you would like to see improved, then posting the suggested change for that page in the main ticket comment, as close to the result as you would like to see it as a tech reader or IT manager.

You could also suggest changes more directly by creating Pull Requests, if you are aware of what to change, where to put the changes and exactly how to formulate the new content.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
